src/libs/actions/Report.js

```diff
@@ -538,7 +538,12 @@ function setNewMarkerPosition(reportID, sequenceNumber) {
 function updateReportActionMessage(reportID, sequenceNumber, message) {
     const actionToMerge = {};
     actionToMerge[sequenceNumber] = {message: [message]};
-    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, actionToMerge);
+    Onyx.merge(`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`, actionToMerge).then(() => {
+        // If the message is deleted, update the last read in case the deleted message is being counted in the unreadActionCount
+        if (!message.text) {
+            setLocalLastRead(reportID, lastReadSequenceNumbers[reportID]);
+        }
```

Comment:
is this what the linter is looking for?

```suggestion
        if (message.text) {
            return;
        }
        setLocalLastRead(reportID, lastReadSequenceNumbers[reportID]);
```